corehq/apps/translations/templates/translations/base_template.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'hqwebapp/base_section.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block page_navigation %}
+    <h2 class="text-hq-nav-header">{% trans "Manage Translations" %}</h2>
+    <ul class="nav nav-hq-sidebar">
+        <li>
+            <a href="{% url 'convert_translations' domain %}">
+                {% trans "Convert Translations" %}
+            </a>
+        </li>
+        <li>
+            <a href="{% url 'icds_app_translations' domain %}">
+                {% trans "Transifex Integration" %}
+            </a>
+        </li>
+    </ul>
```

Comment:
Yes, please - looking at `ConvertTranslations` at least it looks like you already have `page_title` and `section_name` set, so it might just work or you might need to add in `parent_properties` or something like that.

Reply:
@orangejenny @esoergel not sure whats the change needed here. 
Also i have not added this as/under a tab anywhere yet in html since all of this is still for custom. Also was not sure where to add this in dropdowns available but under app seems like the best places for now and hence the urls are under `/a/<domain>/apps/`. So for now thought to keeping it a secret url that is not available as a link on HQ.

Reply:
Sorry, I misread Ethan's comment. He & I are talking about two different things, but they're both about making the UI code more standard.

Even though this is custom, it's still better to add to a tab, so long as it's hidden behind the appropriate toggle/settings, rather than having bespoke UI code and secret URLs (people are bad at remembering them). I think that you can add a new tab with the `apps/translations` prefix and as long as you don't add it to [MENU_TABS](https://github.com/dimagi/commcare-hq/blob/master/corehq/tabs/config.py) it won't show in the top nav. Then you could add a general "Translations" link in the apps menu, probably under "New Application". You'll also need this template to extend `two_column.html` so the sidebar is created for you.

I was talking about the breadcrumbs, which get set based on `section_name`, `parent_pages`, etc.

Reply:
Yeha, I was talking about the sidebar which is typically declared in that `tabclasses.py` file and automatically generated in `two_column.html`.  That basically forms a site map of all of HQ.

Here's a recent example where I made a report appear on the domain admin page when the corresponding feature flag was enabled:
https://github.com/dimagi/commcare-hq/blob/075bfbb5bfe358eb4fffaef4c8df2330e0300009/corehq/tabs/tabclasses.py#L1585-L1590

Note also that the dropdown menu is also controlled by this interface - you can add these things there by adding `'show_in_dropdown': True`

Reply:
Thanks both,
[here](https://github.com/dimagi/commcare-hq/pull/21516/commits/f7b994e61c68b6f046f9569714fd488461fd412e) is my attempt for this.
I have added this as a separate link under Applications Tab if the feature flag for transifex integration is enabled which is at a user level.

Couple of things:
1. Added the link under "Applications" tab for translations.
2. The applications tab is not highlighted when visiting these pages which i think should be expected. I think the only way around is to add another tab as "Translations" and handle its view permission (but "Translations" itself does not look like something that should be a tab) or to edit the ApplicationsTab itself itself instead of adding a new tab. Should that be needed or its okay for now?

Reply:
That looks good to me.

> The applications tab is not highlighted

I thought you could do this by making `url_prefix_formats` something like `/a/{domain}/apps/translations/` (and updating urls.py to correspond, of course), but if you've already futzed with this and it isn't working, I think it's fine that the tab isn't highlighted.

Reply:
I tried that. If i do that the `ApplicationTab` becomes the active tab and the `TranslationsTab` does not get picked and hence the side bar is not added.